cvat-canvas3d/README.md

```diff
@@ -0,0 +1,248 @@
+# Module CVAT-CANVAS-3D
+
+## Description
+
+The CVAT module written in TypeScript language.
+It presents a canvas to viewing, drawing and editing of annotations.
```

Comment:
3D annotations

Reply:
Please double check this file that it's corresponds in particularly for canvas3D module